_posts/data-announces-formation.md

```diff
@@ -55,8 +61,8 @@ Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org)
 Phone: (214) 555-1234  
```

Comment:
I know this has been talked about but just a reminder to remove this if we don't have a number to use

Reply:
Phone number removed.

Reply:
Why did you revert the line break I added?

Reply:
That might have been unintentional. Apologies